src/compiler.js

```diff
@@ -124,7 +124,7 @@ var Compiler = Object.extend({
     },
 
     _templateName: function() {
-        return this.templateName == null? 'undefined' : '"'+this.templateName.replace(/"/g, '\\"')+'"';
+        return this.templateName == null? 'undefined' : JSON.stringify(this.templateName);
```

Comment:
Because we need to escape the `templateName` to be work in a javascript string, I think it's most secure way to do it, we need to handle `\`, `"`, `'`, ...
By using `.replace`, we may forget one case or not handle the combinaison.

What is you suggestion?